S/Singular/build_tarballs.jl

```diff
@@ -56,7 +56,7 @@ products = [
 dependencies = [
     Dependency("cddlib_jll"),
     Dependency("FLINT_jll"),
-    Dependency("GMP_jll"),
+    Dependency("GMP_jll", v"6.1.2"),
```

Comment:
I'm not sure this is the right solution though, wouldn't this make the package uninstallable in Julia v1.6?

Reply:
As I understood @staticfloat this only is a build time version restriction, it does *not* affect the compat version for GMP_jll in Project.toml (for that one still needs to use a PkgSpec as before)

Reply:
Right, this is different from specifying the version in `PkgSpec`!